packages/scripts/formatFiles.js

```diff
@@ -4,9 +4,10 @@
  */
 'use strict';
 
-/// These constants control which files are formatted
+/// These constants control which files are formatted. Be sure to add updates
+/// to ".lintstagedrc.json" in the project root.
 const includeEndsWith = ['.h', '.cpp'];
-const excludePathContains = ['cppwinrt/winrt'];
+const excludePathContains = [];
 const excludePathEndsWith = ['.g.h', '.g.cpp'];
```

Comment:
>.g.h [](start = 30, length = 4)

Shouldn't these be already formatted properly? Why are they excluded? Our own tools are generating these files, so we have full control and they should follow proper format, thus they shouldn't be excluded.

Reply:
I'm not sure of the original motivation, but I could imagine it being something like what we do internally for automatic mock generation. E.g. something generates a header during the build, and will fail the build if the generated header doesn't look like the current one on disk. We could try to teach all of our tools to run clang-format like you mention, but I could totally expect their to be some external tools generating things. Thinking about things like editing an IDL file for instance.

Reply:
For externally generated files yes, but these are not external, so they should adhere to whatever C++ formatting we define.